tests/ui_tests/__init__.py

```diff
@@ -99,10 +103,18 @@ def _process_tested(fixture_test_path: Path, test_name: str) -> None:
 
 @contextmanager
 def screen_recording(
-    client: Client, request: pytest.FixtureRequest
+    client: Client, request: pytest.FixtureRequest, model: str
```

Comment:
why not extract model from the `client`?

Reply:
Surely, I missed we have access to it